src/current/v24.1/work-with-virtual-clusters.md

```diff
@@ -114,7 +114,7 @@ sql_txn_commit_count{tenant="demo"} 0
 
 When connected to a virtual cluster from the DB Console:
 
-- Most pages and views are scoped to a virtual cluster. By default the DB Console displays only metrics about that virtual cluster, and excludes metrics for other virtual clusters and the system virtual cluster. To allow the DB Console to display system-level metrics from within a virtual cluster, you can grant the virtual cluster the `can_view_node_info` permission.
+- Most pages and views are scoped to a virtual cluster. The DB Console displays only metrics about that virtual cluster, and excludes metrics for other virtual clusters and the system virtual cluster.
 
 - DB Console pages related to SQL activity and jobs are visible only from a virtual cluster.
```

Comment:
Is this still true?

Reply:
I added a bit to clarify this. I think what this bullet is trying to imply is that you can't see the virtual cluster stuff from the system cluster (whereas with metrics, the system cluster **can** see different tenant metrics)

Reply:
Yes. The spirit of this statement is that this info is **not** visible in the system cluster (while virtual cluster metrics **are** visible in the system cluster)